src/cider/nrepl/middleware/test.clj

```diff
@@ -200,13 +200,15 @@
 (defn test-vars
   "Call `test-var` on each var, with the fixtures defined for namespace
   object `ns`."
-  [ns vars]
+  [ns test-selector vars]
```

Comment:
I guess this should be just `selector`, as it can't really be a non test selector here. :-) You'll also have to update all the docstrings to mention this.